src/Styles/FeaturedProductsStyles.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+import { device } from "../utils/viewports";
+
+export const FeaturedProductsContainer = styled.section`
+  margin-top: 15px;
+`;
+export const GridContainer = styled.div`
+  display: grid;
+  column-gap: 1rem;
+  row-gap: 1.5rem;
+
+  ${device.mobile} {
+    grid-template-columns: repeat(1, 1fr);
```

Comment:
I noticed you're using flexbox and grid at the same time, I'd recommend choosing one.